### ISSUE TITLE
posibility of a 2.3.4 release with less strict cheerio version dependency 

### ISSUE THREAD

**Issue by evansrobert** [voltidev/ember-svg-jar#197]
Hi, @jherdman @ivanvotti, I'd like to report a vulnerability introduced by package **css-what**:
### Issue Description

   I noticed that a vulnerability is introduced in **_ember-svg-jar@2.3.3_**:
     Vulnerability **CVE-2021-33587** affects package **_css-what_** (versions:<5.0.1):  [https://snyk.io/vuln/SNYK-JS-CSSWHAT-1298035](https://snyk.io/vuln/SNYK-JS-CSSWHAT-1298035)
  The above vulnerable package is referenced by **_ember-svg-jar@2.3.3_** via:
`ember-svg-jar@2.3.3 ➔ cheerio@0.22.0 ➔ css-select@1.2.0 ➔ css-what@2.1.3`

Since **_ember-svg-jar@2.3.3_** ([26,294 downloads per week](https://www.npmjs.com/package/ember-svg-jar/v/2.3.3?activeTab=versions)) is referenced by **25** downstream projects (e.g., [ember-cli-addon-docs 3.0.0](https://www.npmjs.com/package/ember-cli-addon-docs) (latest version), [@freshworks/button 0.18.0](https://www.npmjs.com/package/@freshworks/button) (latest version), [@freshworks/icon 0.20.0](https://www.npmjs.com/package/@freshworks/icon) (latest version), [@freshworks/toast-message 0.18.0](https://www.npmjs.com/package/@freshworks/toast-message) (latest version), [@hashicorp/pds-ember 0.6.2](https://www.npmjs.com/package/@hashicorp/pds-ember) (latest version)), the vulnerability [**CVE-2021-33587**](https://snyk.io/vuln/SNYK-JS-CSSWHAT-1298035) can be propagated into these downstream projects and expose security threats to them via the following package dependency paths:
 (1)`@cardstack/boxel@0.19.42 ➔ ember-svg-jar@2.3.3 ➔ cheerio@0.22.0 ➔ css-select@1.2.0 ➔ css-what@2.1.3`
(2)`@ember-eui/changeset-form@0.7.15 ➔ @ember-eui/core@0.7.15 ➔ ember-svg-jar@2.3.3 ➔ cheerio@0.22.0 ➔ css-select@1.2.0 ➔ css-what@2.1.3`
  **......**

If **_ember-svg-jar@2.3.\*_** removes the vulnerable package from the above version, then its fixed version can help downstream users decrease their pain.

**Given the large number of downstream users, could you help update your package to remove the vulnerability from ember-svg-jar@2.3.3 ?**

### Fixing suggestions

In **_ember-svg-jar@2.3.4_**, maybe you can kindly try to perform the following upgrade :
`cheerio ^0.22.0 ➔ ^1.0.0-rc.4`; 

**Note:**
_**cheerio@1.0.0-rc.4**(>=1.0.0-rc.4) doesn’t depends on **css-what** any more._

Thank you for your attention to this issue and welcome to share other ways to resolve the issue.

Best regards,
^_^

**Comment by jherdman** [comment]
Hi @evansrobert. I'm sure we can do something here. Are you interested in opening a PR that does the work to upgrade the dependency?

**Comment by evansrobert** [comment]
@jherdman Thanks.

**Comment by grantyang** [comment]
Hi @jherdman, is there an expected timeline for merging in the dependency update to resolve this issue? Thanks!

**Comment by jherdman** [comment]
Hi @grantyang . I've poked a bit at trying to resolve this, but I've ran into some difficulties coming to the solution and finding the time to implement it. The timeline is more or less "some day, hopefully soon."

A pull request would be greatly appreciated if anyone has the time and interest.

I confess that I'm not really thrilled at the idea of adopting a modern version of cheerio when it's been in beta for months on end and seems to have been stalled out. I'd love to see an alternative adopted that is much more stable, though I haven't identified one at this time.
